src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -54,7 +55,47 @@ public sealed class CSharpCompilationOptions : CompilationOptions, IEquatable<CS
             StrongNameProvider strongNameProvider = null)
             : this(outputKind, false, moduleName, mainTypeName, scriptClassName, usings, optimizationLevel, checkOverflow, allowUnsafe,
                    cryptoKeyContainer, cryptoKeyFile, cryptoPublicKey, delaySign, platform, generalDiagnosticOption, warningLevel,
+                   specificDiagnosticOptions, concurrentBuild, deterministic,
+                   extendedCustomDebugInformation: true,
```

Comment:
Not sure. I don't think we need extendedCustomDebugInformation -- I think it was added as a mitigation of changes to the PDB CDI records implemented for EnC and causing CCI to break, but a) we haven't exposed this boolean thru command line/project system, so customers can't use it and b) no one actually asked for it. //cc @cston